daprdocs/data/components/middleware/http.yaml

```diff
@@ -24,22 +24,22 @@
   version: v1
   description: Applies [Rego/OPA Policies](https://www.openpolicyagent.org/) to incoming Dapr HTTP requests
 - component: Sentinel
-  link: middleware-sentinel
+  link: /reference/components-reference/supported-middleware/middleware-sentinel
```

Comment:
Pretty sure that these is not supposed to be direct links to the docs. For example the https://docs.dapr.io/reference/components-reference/supported-middleware/middleware-sentinel/ link works currently sentinel and this is just "link: middleware-sentinel" rather than link: /reference/components-reference/supported-middleware/middleware-sentinel